cmd/agent/app/integrations_darwin.go

```diff
@@ -8,19 +8,27 @@
 package app
 
 import (
+	"fmt"
 	"path/filepath"
 )
 
 const (
-	pythonBin = "python2"
+	pythonBin = "python"
 )
 
-var (
-	relPyPath              = filepath.Join("..", "..", "embedded", "bin", pythonBin)
-	relChecksPath          = filepath.Join("..", "..", "embedded", "lib", "python2.7", "site-packages", "datadog_checks")
-	relReqAgentReleasePath = filepath.Join("..", "..", reqAgentReleaseFile)
-	relConstraintsPath     = filepath.Join("..", "..", constraintsFile)
-)
+func getRelPyPath() string {
```

Comment:
Is there a reason for this not to return the absolute path? Looks we compute the absolute path by joining with `root` where this is used

Reply:
root is not found during init so may not be set yet